channels/OFF-CHAIN.md

```diff
@@ -68,12 +68,23 @@ The following message codes are defined:
 | channel_open     | 1  |
 | channel_accept   | 2  |
 | channel_reestabl | 3  |
-| funding_created  | 4  |
-| funding_signed   | 5  |
-| funding_locked   | 6  |
-| update           | 7  |
-| update_ack       | 8  |
-| update_error     | 9  |
+| channel_reest_ack| 4
```

Comment:
I see codes match the [epoch ones](https://github.com/aeternity/epoch/blob/master/apps/aechannel/src/aesc_codec.hrl#L30). Ok